src/network/networkctl.c

```diff
@@ -119,7 +119,7 @@ static int check_netns_match(void) {
 
         if (id == 0) {
                 log_debug("systemd-networkd.service not running in a network namespace (?), skipping netns check.");
-                return 0;
+                goto pass;
```

Comment:
why use goto? just indent what comes next with and `else` block?

goto is ok to use, but if it's easy to avoid I'd still avoid it...